src/events.js

```diff
@@ -2894,9 +2931,9 @@ const displayResearchSpecimenCollectedModal = async (participantData) => {
     let template;
 
```

Comment:
I agree with Amber that there's no need to check here, since code returns `null` if all of them are falsy. If code runs here, it means at least one of the values is truthy.